src/Pokemon.java

```diff
@@ -0,0 +1,42 @@
+abstract public class Pokemon {
+    private String name = "";
+    private int level = 0;
+    private int hp = 0;
+    private String food = "";
+    private String sound = "";
```

Comment:
4 van deze 5 variabelen mogen het "final" keyword krijgen. Dat betekend dat ze 1 keer in gezet mogen worden (in de constructor meestal) en daarna niet meer aangepast mogen worden. De final-variabelen hoeven dus ook geen setters te krijgen